src/Microsoft.AspNetCore.OData/MetadataController.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.OData.Common;
+using Microsoft.AspNet.OData.Extensions;
+using Microsoft.OData.Edm;
+using Microsoft.OData.Edm.Csdl;
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// Represents a controller for generating OData servicedoc and metadata document ($metadata).
+    /// </summary>
+    public partial class MetadataController
```

Comment:
Why Partial class?

Reply:
It had to do with namespaces at one point. This line:

IEdmModel model = Request.GetModel();

Is implemented as an extension method. Request can either be HttpRequestMessage (AspNet) or HttpRequest (AspNetCore). I've tried to keep references to platform-specific code in a platform-specific file. This matter a lot more when the extension methods were in different namesapces.

Since aligning the namespaces, that bit of code can execute correctly in either project. As a result, I will get rid of the partial class.